src/client/user/components/Drawer/ControlPanel/index.tsx

```diff
@@ -131,6 +157,11 @@ export default function ControlPanel() {
     drawerStates.relevantShortLink!,
   )
 
+  // Toggle Link History
+  const isHistoryToggled = drawerStates.linkHistoryIsToggled
+  const toggleHistory: () => void = () =>
+    modalDispatch({ type: DrawerActions.toggleLinkHistory })
```

Comment:
nit: change to `isLinkHistoryToggled` and `toggleLinkHistory`?

Reply:
changed for better readability 👍